### PR TITLE
Update and rename protocol.md to shakeholdersUA.md

### PR DESCRIPTION
**Pull request by Koroliuk**

### CODE REVIEW

**Review by a-mountain — APPROVED** [review]
OK

File: docs/shakeholdersUA.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 1. [Вступ](#1-вступ)  
 1.1 [Мета](#11-мета)  
 1.2 [Контекст](#12-контекст)  
+1.3 [Перелік скорочень та визначень](13#-перелік-скорочень-та-визначень)
```
Comment (Pienskoi) [inline docs/shakeholdersUA.md]:
```suggestion
1.3 [Перелік скорочень та визначень](#13-перелік-скорочень-та-визначень)  

```

**Review by Pienskoi — CHANGES_REQUESTED** [review]

File: docs/shakeholdersUA.md (hunk 1 above)
Comment (Koroliuk) [reply docs/shakeholdersUA.md]: My bad)

**Review by Proxima-C — APPROVED** [review]
Ok 👍

**Review by Pienskoi — APPROVED** [review]
👍

**Pull request merged**